gradle-baseline-java/src/main/java/com/palantir/baseline/plugins/BaselineModuleJvmArgs.java

```diff
@@ -233,13 +234,13 @@ public void execute(Manifest manifest) {
     }
 
     private static ModuleJvmArgsArgumentProvider newModuleJvmArgsArgumentProvider(
-            Project project, BaselineModuleJvmArgsExtension extension, Object classpath) {
+            Project project, BaselineModuleJvmArgsExtension extension, Object classpath, String taskName) {
```

Comment:
Might be better for callers to wrap values in `project.files(...)` to get some better type-safety here.

```suggestion
            Project project, BaselineModuleJvmArgsExtension extension, ConfigurableFileCollection classpath, String taskName) {
```

Reply:
yep good idea updated